using-etherscan.md

```diff
@@ -0,0 +1,24 @@
+# Using etherscan for Linnia Events
+
+# Intro
+
+Etherscan is a useful third party tool that can be used to confirm testnet transactions.
+
+The following is a good general introduction to etherscan.io.
+https://www.youtube.com/watch?time_continue=449&v=7yAxcKabNNM
```

Comment:
The link goes directly to the end of the video. Suggest changing the URL to: https://www.youtube.com/watch?v=7yAxcKabNNM